apidoc/Titanium/UI/Toolbar.yml

```diff
@@ -32,20 +55,36 @@ properties:
 
         For information about color values, see the "Colors" section of <Titanium.UI>. 
     type: String
+    platforms: [iphone, ipad]
     
   - name: items
     summary: An array of buttons (or other widgets) contained in the toolbar.
     type: Array<Titanium.UI.View>
+    platforms: [iphone, ipad]
     
   - name: borderTop
-    summary: If `true`, a border is drawn on the top of the toolbar.
+    summary: If `true`, a border is drawn on the top of the toolbar. This property is ignored on iOS 7 and above.
```

Comment:
As this property do not have effect in iOS 7 and above and we are supporting iOS 8 and above, deprecate it .